force-app/main/default/chat/lwc/messageItem/messageItem.html

```diff
@@ -0,0 +1,28 @@
+<template>
+    <lightning-layout vertical-align="center" horizontal-align="spread">
+        <lightning-layout-item class="user-icon" flexibility="auto" size="1">
+            <lightning-icon icon-name="utility:user"
+                            size="small"
+                            title="User Icon"
+                            alternative-text="User Icon"
+            ></lightning-icon>
+        </lightning-layout-item>
+        <lightning-layout-item flexibility="auto" size="11">
+            <div>
```

Comment:
I would suggest adding some class value for this div for readability

Reply:
fixed 8d870610e229cc8a420aa5daeddcdb7010ca08a5